ros-gazebo_plugins/install.bash

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+
+if [ "$TUE_ROS_DISTRO" != "melodic" ] && [ -L "$TUE_SYSTEM_DIR"/src/gazebo_plugins ]
+then
+    tue-install-info "Removing gazebo_plugins from workspace, using apt version again"
+    rm "$TUE_SYSTEM_DIR"/src/gazebo_plugins
+    catkin clean --workspace "$TUE_SYSTEM_DIR" --orphans
+fi
```

Comment:
Is er geen betere manier om dit te verwijderen? Zou dit niet automagisch moeten gaan met tue-get update?  En zo niet, waarom maken we dat niet?

Reply:
Nee die manier is er niet. Opzich komt zo'n situatie niet vaak voor. En zeker omdat dit tijdelijk is. Houd ik het liever zo.

Reply:
Why not remove the repo too?